### DISCUSSION TITLE
I'm trying to train this model, but I'm having the following problem and am asking for help

### DISCUSSION THREAD

**Discussion by Tranbaber** [apple/ml-cvnets#96] [Q&A]
File "C:\Users\72344\.conda\envs\MobileViTv2\Scripts\cvnets-train.exe\__main__.py", line 4, in <module>
ModuleNotFoundError: No module named 'main_train'

And I tried to download this module, but show
"ERROR: Could not find a version that satisfies the requirement main_train (from versions: none)
ERROR: No matching distribution found for main_train"

What can I do?

**Comment by acrlife** [comment]
I also have this question,are you have a solution?

**Reply by Tranbaber** [reply]
@acrlife 
Write py_modules=["main_train", "main_eval", "main_conversion", "main_loss_landscape"],
in setup.py before entry_points as an argument in the setup() function. and then run pip install --editable . to reinstall the package with these changes.
Good Luck

**Reply by acrlife** [reply]
Thank you!
And,use `export PYTHONPATH=.`  also works!
